### ISSUE TITLE
Add telemetry

### ISSUE THREAD

**Issue by mcomella** [mozilla-mobile/prox#553]
Initial comments: collect [the same data we did in Hawaii](https://docs.google.com/document/d/1JH0RqRp0o76cmsPRLd8Sn8f6fWEfUGjdYxS6RY3rCxI/edit#heading=h.t6lqjkraesjs).

**Comment by brampitoyo** [comment]
I’ve been thinking about solving some problems with analytics in a document called [Detecting Activities](https://docs.google.com/document/d/1BBhszfWWslZhhs-19LSku2S6vsZ3meR-Svw8WkcQAV8/edit).

The gist of my proposal was to correlate some of the data that we’re already planning to collect (under the heading “Optional for Prox v2”) and some new data:

* (Item 11) What individual cards in the Home view are being viewed?
* (Item 12) What cards serve as user session end points? (i.e. cards where they click on a link for directions)?
* (New) How many users ended up actually arriving (being in the proximity of) at a place after tapping “Get directions”?
* (New, but for the future) What places get rated (using third-party app, connected to Prox) after a visit?

TL;DR – We should add telemetry probes for which cards are viewed in the app, and which cards are visited in real life.

By collecting this data, I argued that we’d be able to augment our default sort order. Instead of sorting by proximity, we can prioritise cards with the same same category as:

1. Cards you’ve highly rated in Yelp, TripAdvisor and/or Foursquare
2. Cards you’ve physically been to
3. Cards where you’ve tapped “Get directions”
4. Cards that you’ve spent a lot of time viewing/engaging with
